app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,10 @@
 <resources>
+    <string name="common_ok">OK</string>
```

Comment:
Why not just use the Android built-in resources for these (well except Remove - for that matter why is Remove in here as it's not used)?  Feels like these are unnecessary for the template.